bridge/github/config.go

```diff
@@ -184,6 +194,135 @@ func randomFingerprint() string {
 	return string(b)
 }
 
+func promptTokenOptions(owner, project string) (string, error) {
+	for {
+		fmt.Println()
+		fmt.Println("[0]: user provided token")
+		fmt.Println("[1]: interactive token creation")
+		fmt.Print("Select option: ")
+
+		line, err := bufio.NewReader(os.Stdin).ReadString('\n')
+		fmt.Println()
+		if err != nil {
+			return "", err
+		}
+
+		line = strings.TrimRight(line, "\n")
+
+		index, err := strconv.Atoi(line)
+		if err != nil || (index != 0 && index != 1) {
+			fmt.Println("invalid input")
+			continue
+		}
+
+		if index == 0 {
+			return promptToken()
+		}
+
+		return loginAndRequestToken(owner, project)
+	}
+}
+
+func promptToken() (string, error) {
+	fmt.Println("You can generate a new token by visiting https://github.com/settings/tokens.")
+	fmt.Println("Choose 'Generate new token' and set the necessary access scope for your repository.")
+	fmt.Println()
+	fmt.Println("The access scope depend on the type of repository.")
+	fmt.Println("Public:")
+	fmt.Println("  - 'public_repo': to be able to read public repositories")
+	fmt.Println("Private:")
+	fmt.Println("  - 'repo'       : to be able to read private repositories")
+	fmt.Println()
+
+	for {
+		fmt.Print("Enter token: ")
+
+		byteToken, err := terminal.ReadPassword(int(syscall.Stdin))
```

Comment:
Trying to use it, I don't think masking the input is the way to go for the token. It's not a password that the user is used to type, it's not even something the user will type. What you want to to be able to check that it was pasted properly, that there is no extra spaces or weird character. I think it needs to be visible.

Reply:
I hesitated before writing this, agreed checking the token and making it visible is more suitable for this situation